jsapp/js/actions.es6

```diff
@@ -94,6 +94,12 @@ actions.misc = Reflux.createActions({
   getServerEnvironment: {children: ['completed', 'failed']},
 });
 
+actions.media = Reflux.createActions({
```

Comment:
It would be nice if you'd split media actions to separate file, similar to `jsapp/js/actions/submissions.es6` :heart: